addons/website/tests/test_res_users.py

```diff
@@ -60,12 +60,34 @@ def check_new_test_user(dbname):
         with self.assertRaises(ValidationError), mute_logger('odoo.sql_db'):
             check_new_test_user(self.env.registry._db.dbname)
 
-    def _create_user_via_website(self, website, login):
+    def test_reset_password_user_specific_user_account(self):
+        # Create same user on different websites with 'Specific User Account' option enabled
+        # and then reset password. Only the user from the current website should be reset.
```

Comment:
Nitpicking: can you line return at 80 char max? https://github.com/odoo/odoo/pull/83508#discussion_r796442104